docs/run-keep-ecdsa.adoc

```diff
@@ -0,0 +1,419 @@
+:toc: macro
+
+= Run ECDSA Keep
+
+:icons: font
+:numbered:
+toc::[]
+
+== System Considerations
+
+The Keep Network expects certain capabilites for each node running on the network.  To help attain
+these capabilities consider the following criteria:
+
+- It is paramount that Keep nodes remain available to the Keep Network. We strongly encourage a
+  stable and redundant internet connection.
+- A connection to a production grade self-hosted or third party Ethereum node deployment.
+- Persistent and redundant storage that will survive a VM or container rotation, and disk failure.
+- Each Keep ECDSA client running on the network requires a unique Ethereum operator account.
+- Each Keep ECDSA client running on the network requires a unique IP address or a unique
+  application port running under the same IP.
+- Recommended machine types by provider:
+
+[%header,cols=2*]
+|===
+|Cloud Provider
+|Machine Type
+
+|Google Cloud
+|n1-standard-4
+
+|AWS
+|m5.xlarge
+
+|Azure
+|D4s v3
+
+|Self-hosted
+|4 vCPU / 4 GiB RAM / 1 GiB Persistent Storage
+|===
+
+== Configuration
+
+=== Network
+
+Default port mappings.
+
+[%header,cols=2*]
+|===
+|Egress
+|Port
+
+|Ethereum Network
+|`8545` / `8546`
+
+|Keep Network
+|`3919`
+|===
+
+[%header,cols=2*]
+|===
+|Ingress
+|Port
+
+|Keep Network
+|`3919`
+|===
+
+If you set a different `port` in your keep-ecdsa configuration, or configure `peers` with
+non-default ports configured, firewall rules will need to be adjusted accordingly.
+
+=== Application
+
+Application configurations are stored in a `.toml` file and passed to the application run command
+ with the `--config` flag.
+
+==== Sample
+
+```
```

Comment:
Add toml after start of code fence for syntax highlighting.
```suggestion
``` toml
```

Reply:
Will try!